scripts/jwt_issuer/up.sh

```diff
@@ -14,8 +14,6 @@ jwt-issuer-up() {
 
     sudo chown $USER:$USER -R $JWT_ISSUER_WORKSPACE
 
-    export JWT_TOKEN_ISSUER_URL="http://localhost:3000/token"
```

Comment:
Why have you removed this? We want to export the details of the JWT issuer when we bring it up? I think you've mixed it up with my comment that JWT issuer info shouldn't be exported when we bring up CCF